js/data.js

```diff
@@ -51,41 +51,30 @@ function sparql2GeoJSON(input) {
 }
 
 
+
 function reservesStyle(feature) {
    return {
        fillColor: feature.properties.no > feature.properties.yes ? 'red' : 'green',
        weight: 2,
        opacity: 1,
        color: 'white',
-       fillOpacity: 0.7
+       fillOpacity: getOpacity(feature.properties.totalVoters)
```

Comment:
Opacity should be based on the fraction of people that voted. So it should be something like: (yes+no)/totalVoters